RUMM-2041 Do not produce view event for android tv helper fragments (#862)
DataDog/dd-sdk-android: docs/integrated_libraries_android.md

{{% /tab %}}
{{< /tabs >}}

### Android TV Leanback support package

> If you are building your application for Android TV devices you will usually have to use the `androidx.leanback:leanback` support module. The special Fragment classes
> provided by these module are creating extra RUM View events which are coming from additional helper fragments that they are using under the hood: `androidx.leanback.app.HeadersSupportFragment`,
> `androidx.leanback.app.RowsSupportFragment`. We are providing a special `AndroidTvComponentPredicate` which filters out these extra views by default:

Reviewer: ```suggestion
If you are building your application for Android TV devices, you will usually have to use the `androidx.leanback:leanback` support module. The special Fragment classes
provided by this module create extra RUM View events which come from additional helper fragments `androidx.leanback.app.HeadersSupportFragment` and
`androidx.leanback.app.RowsSupportFragment`. The included `AndroidTvComponentPredicate` filters out these extra views by default:
```
